coalib/results/Diff.py

```diff
@@ -192,6 +192,14 @@ def modified(self):
         return result
 
     @property
+    def has_changes(self):
+        """
+        :return: True if the modified file is not the same when
```

Comment:
It is a property, you don't expect a return value from a property ;) remove that.
Also change the docstring to `True if the modified file is different than the original file, else False`